kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -986,8 +987,14 @@ def submit(host, entity_class, body):
         resp = _POST(host, api_path, body)
 
     if resp.status_code in {200, 201}:
-        result = resp.json()["results"]
-        return result["kf_id"]
+        result = resp.json()
+        if isinstance(result, dict) and "results" in result:
+            return result.get("kf_id")
+        else:
+            raise RequestException(
+                f"Got response code 200/201 but response is not as expected."
+                f"Response was: {pprint(result)}"
```

Comment:
Looks great! How does this message look in the logs? i'm just curious if it would look cleaner if this line was split into two lines? I honestly don't know if it even matters though.... 
```suggestion
                f"Response was:"
                pprint(result)
```

Reply:
how about
```
                f"Got response code 200/201 but response is not as expected."
                f"Response was: \n{pprint(result)}"
```